File: program/find-area-of-a-rectangle/FindAreaOfARectangle.c
#include <stdio.h>

int main(){

    int length, width, area;  
    printf("Enter length\n");
    scanf("%d", &length);
    printf("Enter width\n");
    scanf("%d", &width);
    
    area = length * width;
    printf("Area = %d\n", area);
    return 0;
}

File: program/find-area-of-a-rectangle/find-area-of-a-rectangle.hs
main :: IO ()
main = do 
     putStrLn "Enter length: " 
     input1 <- getLine
     let length = read input1 :: Float

     putStrLn "Enter width: " 
     input2 <- getLine
     let width = read input2 :: Float

     let area = length * width
     putStrLn $ show (area)
